docs/usage.md

```diff
@@ -0,0 +1,111 @@
+# Usage
+
+### Credentials for DNS providers
+
+The _external-dns-operator_ manages external-dns deployments. It creates pods with correct credentials based on the
+provider configuration in the `ExternalDNS` resource. However, it does not provision the credentials themselves, instead
+it expects the credentials to be in the same namespace as the operator itself. It then copies over the credentials into
+the namespace where the _external-dns_ deployments are created so that they can be mounted by the pods.
+
+## Infoblox
+
+Before creating an `ExternalDNS` resource for the [Infoblox](https://www.infoblox.com/wp-content/uploads/infoblox-deployment-infoblox-rest-api.pdf)
+the following information is required:
+
+1. Grid Master Host
+2. WAPI version
+3. WAPI port
+4. WAPI username
+5. WAPI password
+
+Create a secret with the username and password as follows:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: infoblox-credentials
+  namespace: #operator namespace
+data:
+  EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME: # Base-64 encoded username
+  EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD: # Base-64 encoded password
+```
+
+Then create an `ExternalDNS` resource as follows:
+
+```yaml
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: infoblox-example
+spec:
+  provider:
+    type: Infoblox
+    infoblox:
+      credentials:
+        name: infoblox-credentials
+      gridHost: # the grid master host from the previous step. eg: 172.26.1.200
+      wapiPort: # the WAPI port, eg: 80, 443, 8080
+      wapiVersion: # the WAPI version, eg: 2.11, 2.3.1
+  zones: # Replace with the desired hosted zones
+    - "test1.com"
```

Comment:
`Spec.Zones` field is for zone IDs, so better to make them look like IDs rather than domain names.